lua/lvim/lsp/init.lua

```diff
@@ -95,13 +95,11 @@ function M.setup()
   end
 
   if lvim.use_icons then
-    for _, sign in ipairs(lvim.lsp.diagnostics.signs.values) do
+    for _, sign in ipairs(vim.diagnostic.config().signs.values) do
```

Comment:
nice trick! maybe use this as a guard tho

```suggestion
    for _, sign in ipairs(vim.tbl_get(vim.diagnostic.config(), "signs", "values")) do
```